addon/mixins/text-input-mixin.js

```diff
@@ -16,6 +16,10 @@ function handleChanged() {
     }
   }
 
+  if (typeof this.format === 'function') {
```

Comment:
I guess you have to change the `this.format` to `get(this, 'format')` because it can be a function generated by a computed